### DISCUSSION TITLE
Enable dual WAN load balancing for Technicolor TG799vac Xtream (VANT-W)

### DISCUSSION THREAD

**Discussion by fa1rid** [hack-technicolor/hack-technicolor#310] [General]
I have root access to the device. Any idea please how to enable dual WAN load balancing with failover, weight distribution, or per device routing policy. Thanks.

**Comment by LuKePicci** [comment]
There should be multi WAN support from the mwan package in your firmware. Take mwan configuration examples from https://github.com/FrancYescO/tch_firmware_extracted

Alternatively, you can try mwan3 from openwrt but you will need to correctly disable the built in mwan
